ChainReactionTests/Registration.feature

```diff
@@ -1,7 +1,50 @@
-﻿Feature: Registration
-
-Scenario: Verify goto sign up
-	Given I navigate to the page "www.chainreactioncycles.com/signin"
-	And I see the page is loaded
-	When I click on CreateAnAccount Button
-	Then I go to the page "www.chainreactioncycles.com/create-an-account"
+﻿Feature: Registration
+
+Scenario:  Verify Register
+	Given I Navigate to the url "http://www.chainreactioncycles.com/ru/en"
+	And I See the main page is loaded
+
+	When I Click on RubRub dropdown
+	And I select "English" language and click Update button
+	Then I See the main page is loaded
+	
+	When I Click on Sign In on top menu
+	Then I See the Sign In page is loaded
+
+	When I Click on Create Account button
+	Then I See the Create Account page is loaded
+	When I set following parameters on Create Account page
+		| Field | Value |
+		| name  | test  |
+		| pass  | test  |
+	And I Click Apply button
+	Then I See My Account page is loaded
+
+Scenario: Verify Login and Logout
+	Given I start the browser
+	When I Navigate to the url "http://www.chainreactioncycles.com/ru/en"
+	And I See the main page is loaded
+
+	When I Click on RubRub dropdown
+	And I select "English" language and click Update button
+	Then I See the main page is loaded
+
+	When I Click on Sign In on top menu
+	Then I See the Sign In page is loaded
+
+	When I set following parameters on Sign In page
+		| Field | Value |
+		| email  |   |
+		| pass  |   |
+	And I Click Sign In button
+	Then I See the message on Sing In page

```

Comment:
Need to put the exact message so that it will look as following:
Then I See "blah-blah-blah" message on Sing In page